adminPage/adminPageActions.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once '../dbConn.php';
```

Comment:
I would split the functionality of this file into seperate files, then have each form submit to a different file. keeps your code tidier